I_Arduino1.xml

```diff
@@ -74,7 +74,15 @@
   			if (p ~= nil) then p.setLockStatus(lul_device, lul_settings.newTargetValue) end
   			return 4,0
   		</job>
-    </action>     
+    </action>
+	<action>
+      	<serviceId>urn:upnp-org:serviceId:TemperatureSetpoint1</serviceId>
+      	<name>SetCurrentSetpoint</name>
+      	<job>
+      		if (p ~= nil) then p.Setpoint(lul_device, lul_settings.NewCurrentSetpoint)  end
```

Comment:
So why do we have p.SetpointCool()? You do know that I don't know what I'm doing right?